### ISSUE TITLE
Incomplete error message when using sfdx auth:sfdxurl:store

### ISSUE THREAD

**Issue by AllanOricil** [forcedotcom/cli#1298]
### Summary

When running `sfdx auth:sfdxurl:store` with wrong url format, the error message displays an incomplete message. It says
`force://<clientId>:<clientSecret>:<refreshToken>@<loginUrl>`

![Screenshot 2021-11-26 at 22 43 08](https://user-images.githubusercontent.com/55927613/143664341-195a899d-bbaf-4c3d-86ba-cd1ef309795c.png)

But according to the [docs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_auth_sfdxurl.htm#cli_reference_auth_sfdxurl_store) it should also say `force://<refreshToken>@<instanceUrl>`, as it is another option.

### Steps To Reproduce:

1 - create a auth file with this `force://5Aep861nW.ObxIYDklsebQKIpbMOb7FhQu5@syn--vlocdevbas.my.salesforce.com` (don't worry this is invalid)
2 - verify it is missing one of the possible sfdxAuthUrl 

### Expected result

When the given sfdxAuthUrl is wrong, it should display an error message showing both possible sfdxAuthUrl formats:

force://<refreshToken>@<instanceUrl>
force://<clientId>:<clientSecret>:<refreshToken>@<instanceUrl>

### Actual result

When the given sfdxAuthUrl is wrong, the error message displays force://<clientId>:<clientSecret>:<refreshToken>@<instanceUrl> only

### System Information

````bash
sfdx-cli/7.127.0 darwin-arm64 node-v12.22.1
 allanoricil    sf-metadata   main ≡  +1   12.22.1  羽316ms⠀   sfdx plugins --core                                                                                               zsh  0  22:48:56 
@oclif/plugin-autocomplete 0.3.0 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-help 3.2.5 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.10.2 (core)
@oclif/plugin-update 1.5.0 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
alias 1.1.22 (core)
apex 0.3.0 (core)
auth 1.7.5 (core)
config 1.2.47 (core)
custom-metadata 1.0.12 (core)
data 0.6.4 (core)
generator 1.2.1 (core)
limits 1.2.3 (core)
org 1.9.0 (core)
salesforce-alm 53.4.2 (core)
schema 1.0.10 (core)
sfdx-cli 7.127.0 (core)
sfdx-git-delta 4.10.0
source 1.4.0 (core)
telemetry 1.2.8 (core)
templates 52.6.0 (core)
trust 1.0.11 (core)
user 1.5.2 (core)

````
### Additional information
N/A

**Comment by AllanOricil** [comment]
The docs are also not mentioning if the INSTANCE_URL should or not be with https. It seems it is not necessary because the sfdxAuthUrl that we get when we run `sfdx force:org:display --verbose --json` does not have https.

**Comment by jshackell-sfdc** [comment]
Hi @AllanOricil .  I played around with this a bit, and I believe the docs are wrong: `force://<refreshToken>@<instanceUrl>` is NOT a valid option.   Which means the first part of the error message is correct.  I'll update the docs accordingly.  I also agree that the latter part of the error could be more clear about when to use a protocol or not, so we'll be updating both the error and docs soon.  Thx for the input!

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-10238649](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000TxDv3YAF/view)

**Comment by cristiand391** [comment]
This should be fixed  in latest sfdx v7.130.1 🎉 
Release notes: https://github.com/forcedotcom/cli/tree/main/releasenotes/sfdx#71301-dec-9-2021-stable
